blog/progress-report-feb-2023/index.mdx

```diff
@@ -163,6 +163,12 @@ Another new renderer for Jak 2 -- this one handles all of the glowing light effe
 
 <ReactPlayer controls url={"https://www.youtube.com/watch?v=iznKhvzevAI"} className="blog-video"/>
 
+### Warp Gates <PRLink href="https://github.com/open-goal/jak-project/pull/2244"/>
+
+Warp gates are now functional.
+
+<ReactPlayer controls url={"https://user-images.githubusercontent.com/2515356/221090385-531845e2-3688-4c9e-9773-6b12dc7016b8.mp4"} className="blog-video"/>
```

Comment:
I can upload to youtube if thats preferred, just lmk

Reply:
I'll download it just so it doesn't vanish -- don't worry about it